docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -486,12 +513,14 @@ If the updated values do not fit your use case, you can [override them](/x/react
      formatDensity='spacious'
      selectedSections={selectedSections}
      onSelectedSectionsChange={onSelectedSectionsChange}
+     inputRef={inputRef}
```

Comment:
Did I get this wrong, or should this have a `-` in front of it to show the diff change?

Reply:
It should not
This section says that if you really want to still access those props on the field, then you can duplicate what you pass to the picker and also pass it to the field with `slotProps.field`.
It's clearly not something a lot of people should do